.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1 +1,25 @@
-**Please refer to the contribution guidelines in the README when submitting PRs.**
+<!-- Please refer to the contribution guidelines before raising a PR: https://github.com/localstack/localstack/blob/master/CONTRIBUTING.md -->
+
+<!-- Why am I raising this PR? -->
+## Motivation
+
+
+<!-- What notable changes does this PR make? -->
+## Changes
+
+
+<!-- The following sections are optional, but can be useful! 
+
+## Testing
+
+Description of how to test the changes
+
+## TODO
+
+What's left to do:
+
+- [ ] ...
+- [ ] ...
+
```

Comment:
```suggestion

## Related

<!-- Issues, PRs or external links to give more context to this PR. -->
```

Reply:
i would argue that these should go into the motivation section

Reply:
added a comment :+1: